### PR TITLE
add a project file for Sublime Text, also .astylerc

### PR DESCRIPTION
**Pull request by sparr**
This project file specifies the CDDA repo folder and ~/.cataclysm-dda/ as folders to open, gives astyle settings, and specifies four build systems for running make with different parameters

### CODE REVIEW

**Comment by DeadLeaves** [comment]
Love this. Sublime text is by far my texteditor. Gonna download this personally regardless of it making it into the master.

**Comment by sparr** [comment]
@DeadLeaves the project won't change anything about simply editing files. It just gives you some shortcuts for compiling, options for the astyle package, and a couple of default directories to open.

File: Cataclysm-DDA.sublime-project [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+			"working_dir": "${folder:${project_path:${file_path}}}"
+		}
+	],
+	"folders":
+	[
+		{
+			"path": "."
+		},
+		{
+			"path": "/home/sparr/.cataclysm-dda"
```
Comment (remyroy) [inline Cataclysm-DDA.sublime-project]: The project file should not include a path to a specific user directory.

Comment (sparr) [inline Cataclysm-DDA.sublime-project]: I swear that had a tilde in it when I wrote it. It got expanded somewhere along the way.

**Pull request merged**